<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtVoronoiClass</class>
 <widget class="QMainWindow" name="QtVoronoiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtVoronoi</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Plot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1041</width>
      <height>731</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buildVD">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Build VD</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>220</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>340</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="motionButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Motion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawPoints">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>140</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw points</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1183</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plot</class>
   <extends>QWidget</extends>
   <header location="global">plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>QtVoronoiClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>967</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
